worldeditadditions_commands/commands/selectors/scloud.lua

```diff
@@ -0,0 +1,35 @@
+-- ███████  ██████ ██       ██████  ██    ██ ██████
+-- ██      ██      ██      ██    ██ ██    ██ ██   ██
+-- ███████ ██      ██      ██    ██ ██    ██ ██   ██
+--      ██ ██      ██      ██    ██ ██    ██ ██   ██
+-- ███████  ██████ ███████  ██████   ██████  ██████
```

Comment:
Purely cosmetic lol - makes it look less like a 5 :P
```suggestion
--  ██████  ██████ ██       ██████  ██    ██ ██████
-- ██      ██      ██      ██    ██ ██    ██ ██   ██
-- ███████ ██      ██      ██    ██ ██    ██ ██   ██
--      ██ ██      ██      ██    ██ ██    ██ ██   ██
-- ██████   ██████ ███████  ██████   ██████  ██████
```